### DISCUSSION TITLE
"No module named six" error, but 'six' was installed

### DISCUSSION THREAD

**Discussion by Dragaev** [IronLanguages/ironpython3#1656] [Q&A]
Hi! I try start python script which works with sixgill.pipesim via IronPython. Previously i had an error "No module named sixgill", but after that I was add path to this module and used engine.SetSearchPaths(path). It works for sixgill, but not work for "six" module. Also I tried to uninstall and install "six" module and paste "six" folder to IronPython->Lib. But it still not found. Tell me please why?

**Comment by slozier** [comment]
> paste "six" folder to IronPython->Lib

As far as I know six is a single file (`six.py`) so how did you copy a folder?

**Reply by Dragaev** [reply]
yes
I was copi file six.py to folder ironpython
